classes/02class/exercises/c02-network08/zarajoy/README.md

```diff
@@ -0,0 +1,223 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    inbound is coming into the network. outbound is leaving the network. Just like travelling to the city via freeway, inbound is driving towards the city, outbound is driving away from the city.
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+    NACL allow traffic in and out by default and must be configured to be more restrictive. 
```

Comment:
You sure that this is what happens when you create a new NACL?

Reply:
well now i am not :P will check

Reply:
Yeap. NACLs need to be as restricted as possible, but if you create a new NACL in your VPC now, what will come by defaul? Allow all or Deny All?

Reply:
i still want to say everything is allowed by default when creating a NACL.

Reply:
it only allows everything by default on the default ACLs. If you create a new ACL, it will come without any allow rule, so it will end denying all traffic by default.
Since the exercese is using the default ACLs, you're right that it will allow all traffic. :)